hypothesis-python/tests/cover/test_observability.py

```diff
@@ -32,14 +32,15 @@
 
 
 @seed("deterministic so we don't miss some combination of features")
-@example(a=0, x=4, data=None)
+@example(l=[1], a=0, x=4, data=None)
```

Comment:
added this to cover execution of `status_reason = "exceeded size of current best example"` in `make_testcase`